bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookKeeperServerStats.java

```diff
@@ -42,6 +42,7 @@
     public final static String BOOKIE_ADD_ENTRY = "BOOKIE_ADD_ENTRY";
     public final static String BOOKIE_RECOVERY_ADD_ENTRY = "BOOKIE_RECOVERY_ADD_ENTRY";
     public final static String BOOKIE_READ_ENTRY = "BOOKIE_READ_ENTRY";
+    public final static String BOOKIE_READ_LAST_CONFIRMED = "BOOKIE_READ_LAST_CONFIRMED";
 
```

Comment:
yeah. it might be from a different commit. but it is a bit hard for me to separate that out. If you guys are really concerned about this, I will do that effort. Otherwise, I will leave it here.